src/DGtal/topology/KhalimskySpaceND.h

```diff
@@ -509,7 +610,7 @@ namespace DGtal
      * @return the cell having the topology of [c] and the given
      * digital coordinates [p].
      */
-    Cell uCell( const Point & p, const Cell & c ) const;
+    Cell uCell( Point p, const Cell & c ) const;
```

Comment:
Because the point was anyway copied and modified in the definition. But maybe the previous syntax is better ?